alpha/apps/kaltura/lib/db/KalturaPDO.php

```diff
@@ -113,7 +113,7 @@ protected function getCommentWrapped()
 	/* (non-PHPdoc)
 	 * @see PropelPDO::prepare()
 	 */
-	public function prepare($sql, $driver_options = array())
+	public function prepare($sql, $driver_options = array()): false|PDOStatement
```

Comment:
Don't you get the same deprecation warning on this as well?
https://github.com/kaltura/server/blob/a0e460265edab561825bbdfe9462b316afd5f719/vendor/propel/util/PropelPDO.php

Reply:
Full list is here:
https://kaltura.atlassian.net/browse/VCP-20936?focusedCommentId=1834416 

And I did modify the parent as well (all the way at the end of this pr)